frontend/src/styles/components/MineList.scss

```diff
@@ -0,0 +1,16 @@
+.mine-list table {
+  min-width: 1500px;
+}
```

Comment:
When inspecting the code through dev tools, this line doesn't seem to be taking effect. 

Also, `.ant-table-wrapper` appears to be the same outer div as your custom `.mine-list`, 
might be better to use that as the parent class instead. 

![screen shot 2019-02-04 at 10 38 05 am](https://user-images.githubusercontent.com/25125247/52229565-183ff580-286a-11e9-8924-2ad3a8102af1.png)
![screen shot 2019-02-04 at 10 37 57 am](https://user-images.githubusercontent.com/25125247/52229567-183ff580-286a-11e9-8aef-4a111470cc9b.png)

Reply:
This is specific to the mine-list - I don't think we'd want to assume that every '.ant-table-wrapper' in the application should default to a min-width of 1500px.

The div you are inspecting is sizing as expected in those screenshots. This selector is targeting the TABLE element specifically, not it's encapsulating DIV.

Reply:
two things, 
1) why is this specific to mine-list? We do want a standard across all ant design tables. 
2) When inspecting the css through the dev tools, I didn't see this styling being applied anywhere, that's what led me to believe it wasn't being targeted correctly.

Reply:
I'm not sure. The previous developer had left this table with a hard-coded width of 1500px - I'm just continuing on with that pattern - but leveraging CSS instead of hard-coding the issue (as it was done previously).